apps/designer/app/designer/features/sidebar-left/sidebar-left.tsx

```diff
@@ -45,7 +45,7 @@ export const SidebarLeft = ({ publish }: SidebarLeftProps) => {
   );
 
   return (
-    <Flex>
+    <Flex css={{ flexGrow: 1 }}>
```

Comment:
Doesn't flex have some prop for this?

Reply:
unfortunately no and since flexGrow is a number, its difficult to add using stitches variants. 
i don't like this too, basically with variants you can either predefine all number values or make some kind of grow: true variant that sets to 1, maybe that's what we should do, because that's very popular thing to do

also maybe we should add min-height: 1 and min-width: 1 because that fixes also tons of use cases

Reply:
0 fixes too, no?

Reply:
actually min-height: 0 and min-width 0 also doing the same trick